src/util.ts

```diff
@@ -105,3 +106,14 @@ export const unique = () => {
     return true;
   };
 };
+
+export async function enableCorepack(input: string): Promise<void> {
+  if (input.length && input !== 'false') {
+    const corepackArgs = ['enable'];
+    if (input !== 'true') {
+      const packageManagers = input.split(' ');
+      corepackArgs.push(...packageManagers);
+    }
+    await getCommandOutput(`corepack ${corepackArgs.join(' ')}`);
```

Comment:
Looking back at previous PRs, it was suggested that if Yarn ends up being enabled, any existing Yarn installation should be uninstalled first: https://github.com/actions/setup-node/pull/482#pullrequestreview-1017248195. Is this is a concern?

Reply:
Yes, the existing Yarn installation can be left as is and it won't cause conflict with the installed Yarn from Corepack.